### PR TITLE
About Section: Add a page about contributing and APG TF membership

### LINKED ISSUE #2706: About Section: Add page about contributing with information on joining the group, group communication, and learning how to contribute
I suggest we add information on joining the APG working group and links to our e-mail communication list archive on the "About" page of the APG website.

About Page:
https://www.w3.org/WAI/ARIA/apg/about/

### PR DESCRIPTION
**Pull request by mcking65**
Resolve #2706  by adding a page to the about section about contributing.

### Preview links

* [Preview revised about page](https://deploy-preview-236--aria-practices.netlify.app/aria/apg/about/)
* [Preview new contributing page](https://deploy-preview-236--aria-practices.netlify.app/aria/apg/about/contributing/)

___
[WAI Preview Link](https://deploy-preview-236--aria-practices.netlify.app/ARIA/apg) _(Last built on Mon, 02 Oct 2023 19:57:43 GMT)._

### CODE REVIEW

**Comment by shirsha** [comment]
Please add "Back to top" a landmark role. Discussed in the apg meeting and this will be deleted as it is part of template

The "[W3C Web Accessibility Initiative (WAI)](https://www.w3.org/WAI/)" visually look as heading but not marked up as such

File: content/about/contributing/contributing.html
Comment (daniel-montalvo) [inline content/about/contributing/contributing.html:36]:
```suggestion
        Links to both are provided at the bottom of every APG page.
```

File: content/about/contributing/contributing.html
Comment (daniel-montalvo) [inline content/about/contributing/contributing.html:51]:
```suggestion
        While GitHub issues are the preferred place to discuss APG content, the mailing list is available to anyone who would prefer to communicate by
```

File: content/about/contributing/contributing.html
Comment (daniel-montalvo) [inline content/about/contributing/contributing.html:52]:
```suggestion
        <a href="mailto://public-aria-practices@w3.org">sending email to the APG Task Force</a>.
```

File: content/about/contributing/contributing.html
Comment (daniel-montalvo) [inline content/about/contributing/contributing.html:59]:
```suggestion
    <section id="Making-improvements-to-documentation-or-code">
```

Text to lowercase and hyphenated

File: content/about/contributing/contributing.html
Comment (daniel-montalvo) [inline content/about/contributing/contributing.html:84]:
```suggestion
        <a href="mailto://public-aria-practices@w3.org">sending email to the ARIA Authoring Practices public mailing list</a>.
```

File: content/about/contributing/contributing.html
Comment (daniel-montalvo) [inline content/about/contributing/contributing.html:88]:
```suggestion
    <section id="joining-the-authoring-practices-task-force">
```

File: content/about/contributing/contributing.html
Comment (daniel-montalvo) [inline content/about/contributing/contributing.html:95]:
```suggestion
        The Task Force meets weekly to coordinate work on development of the APG.
```

File: content/about/contributing/contributing.html
Comment (daniel-montalvo) [inline content/about/contributing/contributing.html:96]:
```suggestion
        Task Force participants are expected to regularly attend meetings and to actively contribute.
```

File: content/about/contributing/contributing.html
Comment (daniel-montalvo) [inline content/about/contributing/contributing.html:116]:
```suggestion
    <section id="Helping-provide-assistive-technology-support-data">
```

**Review by daniel-montalvo — CHANGES_REQUESTED** [review]
Thanks for putting all of this together. Minor editorial suggestions.

I would focus mainly on the ids with text to lowercase and hyphenated, for consistency and better predictability of anchor links. I may have skipped some of these

**Review by andreancardona — APPROVED** [review]
Other than Daniel's comments looks good to me!

File: content/about/contributing/contributing.html
Comment (mcking65) [reply content/about/contributing/contributing.html:21]: I commited a similar change. The id you suggested had a misspelling.

File: content/about/contributing/contributing.html
Comment (mcking65) [reply content/about/contributing/contributing.html:24]: I made a similar commit. Your suggestion didn't include the word "Guide".

File: content/about/contributing/contributing.html
Comment (mcking65) [reply content/about/contributing/contributing.html:36]: Left unchanged because so the emphasis is on "every page". "In the APG" is almost superfluous, but I've chosen to leave it because the claim might not be consistently true across all WAI pages.

**Comment by mcking65** [comment]
@shirsha wrote:

> Please add "Back to top" a landmark role. Discussed in the apg meeting and this will be deleted as it is part of template

I don't know if it will be deleted; I would like it to be deleted. However, I don't know if there is a template issue open about it. That'd take a bit of searching.

> The "[W3C Web Accessibility Initiative (WAI)](https://www.w3.org/WAI/)" visually look as heading but not marked up as such

The styling might make it appear to be a heading, but semantically it does not serve as a heading. It is just a prominent link in the banner. It is really important to only use heading elements where the element is semantically serving the purpose of a heading, not just styled in a way that is similar to a heading.

**Comment by mcking65** [comment]
@daniel-montalvo

A quick note on capitalization of the term "task force" since it came up in your review.

I have capitalized the term when it is a proper noun and used lowercase otherwise. That is, when writing out the full name "ARIA Authoring Practices Task Force", the words "Task" and "Force" are capitalized. If the words "task force" appear separate from the full name of the task force, they are not capitalized. I am fairly confident this is consistent with the Chicago manual of style referenced by the editorial guidelines. If there is something else I should consider, please let me know.

File: content/about/contributing/contributing.html
Comment (daniel-montalvo) [reply content/about/contributing/contributing.html:36]: It should be, but I don't feel strongly anyway.

**Comment by daniel-montalvo** [comment]
> @daniel-montalvo
> 
> A quick note on capitalization of the term "task force" since it came up in your review.
> 
> I have capitalized the term when it is a proper noun and used lowercase otherwise. That is, when writing out the full name "ARIA Authoring Practices Task Force", the words "Task" and "Force" are capitalized. If the words "task force" appear separate from the full name of the task force, they are not capitalized. I am fairly confident this is consistent with the Chicago manual of style referenced by the editorial guidelines. If there is something else I should consider, please let me know.

I'm OK as long as we are consistent. What you suggest seems consistent to me.

**Review by daniel-montalvo — APPROVED** [review]
Approving from an editorial perspective. Note that link-checker still flags a broken link  on contributing.html line 69

**Comment by a11ydoer** [comment]
In New contributing page, sending email to APG has two unnecessary slash, //public-aria-practices@w3.org [undefined://public-aria-practices@w3.org] and we can update that in a correct form.

**Review by a11ydoer — CHANGES_REQUESTED** [review]
mailto code should remove two slashes, "//" in front of mailing address.

**Review by a11ydoer — APPROVED** [review]
I fixed mailto code and directly commited the change to speed up the publication. Please let me know if you prefer to creat PR instead despite that this is a minor update, @mcking65

**Comment by mcking65** [comment]
Thank you @a11ydoer for catching that. BTW, the links work with the slashes, at least in Chrome, but they shouldn't be there.

> I fixed mailto code and directly commited the change to speed up the publication.

There were two; I fixed the other.

>Please let me know if you prefer to creat PR instead despite that this is a minor update, @mcking65

Definitely not a PR for minor changes. Using the suggestions feature in a review is the most straightforward way to add code and ensure everyone is aligned, but for something like this where there is no question about it, pushing a commit is fine.

**Comment by mcking65** [comment]
Thank you @a11ydoer and @daniel-montalvo! @howard-e has updated the link-checker in #2812, and I'm just waiting on a review from @alflennik  on that. Then, if all is good, I will merge the link-checker changes. Then, the tests on this should all be passing, and I will merge it.

**Comment by a11ydoer** [comment]
> Thank you @a11ydoer for catching that. BTW, the links work with the slashes, at least in Chrome, but they shouldn't be there.
> 
> > I fixed mailto code and directly commited the change to speed up the publication.
> 
> There were two; I fixed the other.
> 
> > Please let me know if you prefer to creat PR instead despite that this is a minor update, @mcking65
> 
> Definitely not a PR for minor changes. Using the suggestions feature in a review is the most straightforward way to add code and ensure everyone is aligned, but for something like this where there is no question about it, pushing a commit is fine.

Thanks for fixing the error and your feedback about using "suggestions feature"
btw, I tested sending the email with two slashes and outlook reported me the error.

**Comment by shirsha** [comment]
Thanks, Matt !!. Changed changes are reviewed

**Pull request merged**
